enterprise-suite/scripts/lbc.py

```diff
@@ -282,14 +277,6 @@ def are_pvcs_created(namespace):
         fail_msg='Found some PVCs ({}) from previous console install, but not all expected: {}.\nTo avoid data loss, please remove them manually'
     )
 
-# Checks for console cluster roles
-def are_cluster_roles_created():
```

Comment:
Could we still do this test
- if we include namespace in the check query, and
- only do it on envs where it's expected to work?

I guess I'm just asking if there's still some value here.

Reply:
Got it.  Thanks.